core/server/src/main/java/tachyon/web/WebInterfaceDownloadServlet.java

```diff
@@ -69,7 +70,8 @@ public WebInterfaceDownloadServlet(FileSystemMaster fsMaster) {
   @Override
   protected void doGet(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
-    if (PlainSaslServer.AuthorizedClientUser.get() == null) {
+    if (SecurityUtils.isSecurityEnabled(MasterContext.getConf())
```

Comment:
Thanks. Yeah, I understand it. Adding the checking is for showing better error message.